client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/item-row.js

```diff
@@ -0,0 +1,111 @@
+		target="_blank" rel="noopener noreferrer">{ translate( 'look up' ) }</a>)</span>
+);
+
+const ItemRow = ( props ) => {
+	const {
+		errors,
+		packageId,
+		productId,
+		translate,
+		description,
+		defaultDescription,
+		tariffNumber,
+		originCountry,
+		countriesData,
+	} = props;
+
+	return <div className="customs-step__item-row">
+		<TextField
+			id={ packageId + '_' + productId + '_description' }
+			className="customs-step__item-description-column"
+			title={ translate( 'Description' ) }
+			value={ description }
+			placeholder={ defaultDescription }
+			updateValue={ props.setCustomsItemDescription }
+			error={ errors.description } />
+		<CountryDropdown
+			id={ packageId + '_' + productId + '_originCountry' }
+			className="customs-step__item-country-column"
+			title={ translate( 'Origin country' ) }
+			value={ originCountry }
+			updateValue={ props.setCustomsItemOriginCountry }
+			countriesData={ countriesData } />
+		<TextField
+			id={ packageId + '_' + productId + '_tariffNumber' }
+			className="customs-step__item-code-column"
+			title={ <TariffCodeTitle /> }
+			value={ tariffNumber }
+			updateValue={ props.setCustomsItemTariffNumber }
+			error={ errors.tariffNumber } />
+	</div>;
+};
+
+ItemRow.propTypes = {
+	siteId: PropTypes.number.isRequired,
+	orderId: PropTypes.number.isRequired,
+	packageId: PropTypes.string.isRequired,
+	productId: PropTypes.number.isRequired,
+	description: PropTypes.string.isRequired,
+	defaultDescription: PropTypes.string.isRequired,
+	tariffNumber: PropTypes.string.isRequired,
+	originCountry: PropTypes.string.isRequired,
+	errors: PropTypes.object,
+	countriesData: PropTypes.object.isRequired,
+	setCustomsItemDescription: PropTypes.func.isRequired,
+	setCustomsItemTariffNumber: PropTypes.func.isRequired,
+	setCustomsItemOriginCountry: PropTypes.func.isRequired,
+};
+
+const mapStateToProps = ( state, { orderId, siteId, productId } ) => {
+	const loaded = isLoaded( state, orderId, siteId );
+	const shippingLabel = getShippingLabel( state, orderId, siteId );
+	const storeOptions = loaded ? shippingLabel.storeOptions : {};
+
+	return {
+		...shippingLabel.form.customs.items[ productId ],
+		errors: loaded ? getFormErrors( state, orderId, siteId ).customs.items[ productId ] : {},
+		countriesData: storeOptions.countriesData || {},
+	};
+};
+
+const mapDispatchToProps = ( dispatch, { orderId, siteId, productId } ) => ( {
+	setCustomsItemDescription: ( value ) => dispatch( setCustomsItemDescription( orderId, siteId, productId, value ) ),
+	setCustomsItemTariffNumber: ( value ) => dispatch( setCustomsItemTariffNumber( orderId, siteId, productId, value ) ),
+	setCustomsItemOriginCountry: ( value ) => dispatch( setCustomsItemOriginCountry( orderId, siteId, productId, value ) ),
+} );
+
+export default connect( mapStateToProps, mapDispatchToProps )( localize( ItemRow ) );
+
+export const Header = localize( ( { translate } ) => (
```

Comment:
Why not move this to its own file? Also, we currently only import `Header` as `ItemRowHeader` - why not name it `ItemRowHeader`?

Reply:
> Why not move this to its own file?

Because this `Header` component uses the small `<TariffCodeHeader>` component, internal to this file. I didn't want to add 2 more files for 2 5-lines React components.

> Also, we currently only import Header as ItemRowHeader - why not name it ItemRowHeader?

Using the old `const ItemRow = require('./item-row')` would make it clearer, because you would render the component like `<ItemRow.Header/>`. But with `import` it looks more awkward. Changed in https://github.com/Automattic/wp-calypso/pull/24974/commits/1a43fc2364b39bc6b1d168e5a8787fd054d5b1fc.